### PR TITLE
notebook for raw return

### PR DESCRIPTION
**Pull request by ruifanp**

### CODE REVIEW

File: Labelling/Labels Raw Return/Raw Return.ipynb [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+    "ax.set_ylabel(\"frequency\")\n",
+    "ax.set_xlabel(\"Daily Log Return\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also get categorical labels. Note that because on a few days the price did not change at all, np.sign gives a warning for zero values, but this is not actually an error."
+   ]
```
Comment (PanPip) [inline Labelling/Labels Raw Return/Raw Return.ipynb:146]: Please look at the last comment to [PR#357](https://github.com/hudson-and-thames/mlfinlab/pull/357) to remove the Warning message.

File: Labelling/Labels Raw Return/Raw Return.ipynb [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+    "---\n",
+    "## Conclusion"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook presents the raw returns labeling method, which is the most basic and ubiquitous method used in financial data analysis, and is generally surperior to using price changes. In this method, simple or logarithmic returns can be outputted based on price data. This labeling method is often a percursor to more sophisticated labeling methods."
+   ]
```
Comment (PanPip) [inline Labelling/Labels Raw Return/Raw Return.ipynb:276]: `surperior` to `superior`

`percursor` to `precursor`

File: Labelling/Labels Raw Return/Raw Return.ipynb [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+    "# Labeling: Raw Returns\n",
+    "\n",
+    "![image_example](img/Raw_returns_distribution.png)\n",
+    "_*Fig. 1:*_ Distribution of daily logarithmic returns for MSFT between Jan 2010 and May 2020.\n",
+    "\n",
+    "\n",
+    "## Abstract\n",
+    "\n",
+    "Labeling data by raw returns is the most simple and basic method of labeling financial data for machine learning. Raw returns can be calculated either on a simple or logarithmic basis. Using returns rather than prices is usually preferred for financial time series data because returns are usually stationary, unlike prices. This means that returns across different assets, or the same asset at different times, can be directly compared with each other. The same cannot be said of price differences, since the magnitude of the price change is highly dependent on the starting price, which varies with time.\n",
+    "\n",
```
Comment (PanPip) [inline Labelling/Labels Raw Return/Raw Return.ipynb:16]: I'd change `returns are usually stationary, unlike prices` to `returns are closer to being stationary in comparison to raw prices`.

`...the starting price, which varies with time.` - should be changed. Does starting price vary with time?

Comment (ruifanp) [reply Labelling/Labels Raw Return/Raw Return.ipynb:16]: changed starting price to preceding price

**Pull request merged**